server/lib/nl/fulfillment_next.py

```diff
@@ -71,8 +72,16 @@ def fulfill(query_detection: Detection,
   # Each query-type has its own handler. Each knows what arguments it needs and
   # will call on the *_from_context() routines to obtain missing arguments.
   if (uttr.query_type == ClassificationType.SIMPLE):
-    fulfillment_type = 'SIMPLE'
-    simple.populate(uttr)
+    overview_classification = context.classifications_of_type_from_utterance(
+        uttr, ClassificationType.OVERVIEW)
+    if overview_classification and not filtered_svs:
```

Comment:
so "auto theft in plao alto" -> "tell me about san jose" will give an overview page?

Reply:
Yes!